src/common.rs

```diff
@@ -4,9 +4,9 @@ use std::path::PathBuf;
 
 #[derive(Clone, Debug, PartialEq)]
 pub struct Warning {
-    check_name: String,
-    line: LineEntry,
-    message: String,
+    pub check_name: String,
+    pub line: LineEntry,
+    pub message: String,
 }
 
 impl Warning {
```

Comment:
This part is out of control if we just use the output somewhere, which confuses me a little.

I looked a little at the possibilities of `colored` and it seems to me that it will be correct to use [their internal way of configuring the output](https://github.com/mackwic/colored#colorization-control) if it possible.

What do you think? :thinking:

Reply:
Overall, I like the current implementation :+1: 

Now the changes mostly affect only the `main.rs` file and it works obviously.

Reply:
Nice, I'll work on it